x-pack/plugins/index_lifecycle_management/__jest__/components/edit_policy.test.tsx

```diff
@@ -364,10 +398,10 @@ describe('edit policy', () => {
       setPolicyName(rendered, 'mypolicy');
       await activatePhase(rendered, 'warm');
       setPhaseAfter(rendered, 'warm', '1');
-      findTestSubject(rendered, 'forceMergeSwitch').simulate('click');
+      findTestSubject(rendered, 'warm-forceMergeSwitch').simulate('click');
       rendered.update();
-      const shrinkInput = rendered.find('input#warm-selectedForceMergeSegments');
-      shrinkInput.simulate('change', { target: { value: '0' } });
+      const forcemergeInput = rendered.find('input#warm-selectedForceMergeSegments');
```

Comment:
nit; it would be nice to migrate this test (and corresponding component) to use `data-test-subj` instead of HTML id attribute.